app/(gcforms)/[locale]/(form administration)/form-builder/[id]/layout.tsx

```diff
@@ -46,10 +46,6 @@ export default async function Layout({
     if (initialForm === null) {
       redirect(`/${locale}/404`);
     }
-
-    if (initialForm.isPublished) {
```

Comment:
@bryan-robitaille how did you catch this? I ran multiple tests locally and I did not notice anything wrong